package.json

```diff
@@ -84,7 +84,6 @@
     "@types/through2": "^2.0.34",
     "@types/time-span": "^2.0.0",
     "@types/uuid": "^3.4.4",
-    "async": "^2.6.1",
```

Comment:
system-test.ts relies on this along with cleanup.js. We can move it dev dependencies though.